interface/modules/custom_modules/oe-module-ehi-exporter/public/index.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenEMR\Modules\EhiExporter;
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+use OpenEMR\OeUI\OemrUI;
```

Comment:
Should this script be secured by an acl (such as admin/super ?)?

Reply:
Fixed in the updated PR.  Add an ACL check for admin/super.